src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/Filter/search_specification.rst

```diff
@@ -60,7 +60,10 @@ Naming
 ~~~~~~
  - Elasticsearch fields for attribute follow this naming scheme:
 
-``attribute_code-locale-scope-es_suffix``
+``attribute_code-backend_type-locale-scope-es_suffix``
```

Comment:
no...

1. we don't use "scope", we use "channel"
2. `attribute_code-backend_type.channel.locale.es_suffix`